tokio/src/sync/watch.rs

```diff
@@ -114,7 +114,7 @@
 use crate::sync::notify::Notify;
 
 use crate::loom::sync::atomic::AtomicUsize;
-use crate::loom::sync::atomic::Ordering::Relaxed;
+use crate::loom::sync::atomic::Ordering;
```

Comment:
nit, not a big deal: if we had changed this to 
```suggestion
use crate::loom::sync::atomic::Ordering::*;
```
the existing places we use `Relaxed` without prefixing it with the enum name wouldn't have had to change, resulting in a smaller total diff. Not a big deal though.

Reply:
I try to avoid such wildcard imports. But I agree that it makes sense here to avoid altering untouched code.